.github/workflows/python-package.yml

```diff
@@ -28,6 +28,7 @@ jobs:
       run: |
         python -m pip install --upgrade pip
         pip install flake8 pytest
+        pip install Pillow
```

Comment:
lower case p on pilllow?

Reply:
Not according to https://pypi.org/project/Pillow/2.2.2/